tools/ot/src/main/java/dk/alexandra/fresco/tools/ot/otextension/BristolOt.java

```diff
@@ -0,0 +1,75 @@
+package dk.alexandra.fresco.tools.ot.otextension;
+
+import dk.alexandra.fresco.framework.network.Network;
+import dk.alexandra.fresco.framework.util.StrictBitVector;
+import dk.alexandra.fresco.tools.ot.base.Ot;
+
+/**
+ * Container class for a protocol instance of Bristol OTs, which is an OT
+ * extension protocol, secure in the random oracle model.
+ * It is specified in the paper https://eprint.iacr.org/2015/546
+ */
+public class BristolOt implements Ot {
+  private BristolOtSender sender = null;
+  private BristolOtReceiver receiver = null;
+  private final RotFactory rot;
+  private final OtExtensionResourcePool resources;
+  private final Network network;
+  private final int batchSize;
+
+  /**
+   * Constructs a new OT protocol and constructs the internal sender and receiver
```

Comment:
This is not right, these objects are created in the send and receive methods. Possibly the java doc is updated?

Reply:
Updated in 6bd02da5